lxd/images.go

```diff
@@ -281,6 +312,7 @@ func imageDelete(d *Daemon, r *http.Request) Response {
 	}
 
 	_, _ = d.db.Exec("DELETE FROM images_aliases WHERE image_id=(SELECT id FROM images WHERE fingerprint=?);", uuid)
+	_, _ = d.db.Exec("DELETE FROM images_properties WHERE image_id=(SELECT id FROM images WHERE fingerprint=?);", uuid)
```

Comment:
Looks like we'd benefit from caching the image id instead of querying it in both deletes.